msg/MotionSequenceResponse.msg

```diff
@@ -0,0 +1,11 @@
+# An error code reflecting what went wrong
+MoveItErrorCodes error_code
+
+# The full starting state of the robot at the start of the each trajectory-part
+RobotState[] trajectories_start
```

Comment:
Why do you need multiple start states here?
All the trajectories should be continuous, shouldn't they?

Reply:
Yes, the trajectories are continuous but we still return all the start states of the consecutive segments. This is analogous to how `msg/MotionPlanResponse.msg` returns the start state of the trajectory. We just do it multiples times, once per segment of the sequence.

Reply:
Each response requires to report the _full_ initial `RobotState` exactly once. Because the individual trajectory segments only comprise the joint angles of the moved group, we need to have a valid starting point, defining the full state.
The same holds true for a sequence of responses. Where do you see the benefit of sending all individual start states? IMHO this is just redundant here and thus should be avoided.

Reply:
@ct2034, can you please respond to this comment as well?

Reply:
Sorry, I am currently trying to estimate, what the effort would be to change this. I think especially in our python API bigger changes are required.

Reply:
Obviously, the other maintainers didn't care about this issue. Before you start a lot of refactoring, we should explicitly ask @v4hn and @henningkayser about their opinion on this.

Reply:
I would retain the `RobotState`. It allows for changes to the attached objects.
If anything, generalize to `PlanningScene` messages instead.

Reply:
Ok, I agree. In MTC we also have the _final_ PlanningScene state at the end of a trajectory.
This actually raises the question, whether we should join these two messages, i.e. MTC's `Solution` msg and the `MotionSequenceResponse` here. Of course, MTC uses much more detailed information regarding the underlying task.

Reply:
done

Reply:
I agree with you, it makes no sense to keep the redundant states. So I changed it.